fusor-ember-cli/app/models/deployment-plan.js

```diff
@@ -1,8 +1,63 @@
+import Ember from 'ember';
 import DS from 'ember-data';
 
 export default DS.Model.extend({
     name: DS.attr('string'),
     uuid: DS.attr('string'),
     roles: DS.hasMany('deployment-role', {inverse: null, async: false}),
-    parameters: DS.hasMany('deployment-plan-parameter', {inverse: null, async: false})
+    parameters: DS.hasMany('deployment-plan-parameter', {inverse: null, async: false}),
+
+    externalNetworkInterface: Ember.computed('parameters.@each.value', {
+        get(key) {
+            return this.getParamValue('Controller-1::NeutronPublicInterface');
+        },
+        set(key, value) {
+            this.updateParam('Controller-1::NeutronPublicInterface', value);
```

Comment:
@cfchase, nice to use to new syntax for `set` within computed properties. I have used this before. Looks good.